testing/endtoend/minimal_e2e_test.go

```diff
@@ -8,6 +8,6 @@ import (
 )
 
 func TestEndToEnd_MinimalConfig(t *testing.T) {
-	r := e2eMinimal(t, version.Phase0, types.WithCheckpointSync())
+	r := e2eMinimal(t, version.Capella, types.WithCheckpointSync(), types.WithEpochs(6))
```

Comment:
TODO: this needs to be broken out into a separate entrypoint - currently it is shared by pre- and post- submit e2e, but in post-submit we want to continue starting at phase0.